### PR TITLE
Update Opinion interactive, and change atom CAPI URLs to include prefix

### PR DESCRIPTION
**Pull request by katebee**
- The interactives team have updated the URL used for interactives; the changes to each`gulpfile.babel.js` fix the output URL so that it will work in Composer.

- We are no longer planning to bucket new sign ups for Opinion; deleting the randomise JS

- The listID for Opinion has now been set up, so swapping it in

@superfrank 
@joelochlann

### CODE REVIEW

**Comment by katebee** [comment]
@superfrank this has the URL update in it

File: interactive-opinion/src/templates/main.html [hunk 1]
```diff
@@ -16,7 +16,7 @@ <h1 class="email-signup-header__heading">Best of Guardian<span class="coloured-h
                 <div class="email-signup-main">
 
                     <div class="email-signup" data-component="opinion-email-interactive">
-                      <iframe src="https://www.theguardian.com/email/form/plaintone/2313" height="60px" width="100%" scrolling="no" frameborder="0" seamless
+                      <iframe src="https://www.theguardian.com/email/form/plaintone/3811" height="60px" width="100%" scrolling="no" frameborder="0" seamless
```
Comment (katebee) [inline interactive-opinion/src/templates/main.html]: @davidfurey this updates the iframe url to use the 3811... or do you want apps and web to add people to 2313 if JS doesn't run?

Comment (davidfurey) [inline interactive-opinion/src/templates/main.html]: 3811 should be the default if JS doesn't run

**Comment by davidfurey** [comment]
👍

**Pull request merged**